### PR TITLE
CBI-892-outdated-metrics: ignore outdated metrics

### PR DESCRIPTION
**Pull request by naure**

### CODE REVIEW

**Review by evgeny-s — APPROVED** [review]
Metrics rotating still exists, right? But now we have time in value which provides us a possibility to  make sure metrics are not outdated..

File: cere02/lib.rs [hunk 1]
```diff
@@ -1458,20 +1513,25 @@ mod ddc {
...
                     requests: 0,
                 };
                 31
             ];
 
-            expected[17] = MetricValue {
-                stored_bytes,
-                requests,
-            };
+            for i in 0..31 {
```
[the remark this answers is not in the document]
Comment (naure) [reply cere02/lib.rs:1529]: Right, let’s use in tests as well.

File: cere02/lib.rs [hunk 2]
```diff
@@ -651,52 +643,90 @@ mod ddc {
         pub fn metrics_for_period(
             &self,
             app_id: AccountId,
-            start_date_ms: u64,
+            subscription_start_ms: u64,
```
[the remark this answers is not in the document]
Comment (naure) [reply cere02/lib.rs:646]: OCW does not use this function?

**Comment by naure** [comment]
Yes, it’s still the same trick to reuse storage with key/day_of_period, but now checked at read time.

**Pull request merged**